docs/book/common_code/images/sequence_machine_deletion.drawio

```diff
@@ -0,0 +1 @@
+<mxfile modified="2019-03-20T22:28:57.522Z" host="www.draw.io" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36" etag="IE16TxYfpd6yVC7TUXYR" version="10.5.2" type="google"><diagram id="FanLFHlqyqk8pDkD9QlS" name="Page-1">5VrbcpswEP0aPyZjwGD8mDi3h2aaaWbaZxlko0YgIkRs9+u7MuIq3+JgatdPhpW0K+05SEc77lnjcPHIURw8Mx/Tntn3Fz3rrmeahjFy4UdalsrSdweZZcaJr2yl4ZX8wXlHZU2Jj5NaR8EYFSSuGz0WRdgTNRvinM3r3aaM1qPGaIY1w6uHqG79RXwRZFbX7pf2J0xmQR7Z6KuWEOWdlSEJkM/mFZN137PGnDGRPYWLMaYye3lesnEPG1qLiXEciX0GPP2cfl8+RosFuVrOLNt7f54+XCkwPhBN1YLf0gnkkao5i2WeCJh+LB/TkH4jU0xJBG+3MeYkxAJzaKHK/FLabucBEfg1Rp4cOgeOgC0QIYU3Ax4BNYFgCC/eKUVxQiarqH2wcOylPCEf+AdOMnJIK0uFjDQuQJdGtRTMBV5szJFRZB44ixnMky+hixowUFgt66/zEvkC36CC+iDviBTbZoXnEhB4UJh8Ah9bx8dNZKCYJJjDWi8TJ8M5NaAcDagQeYFM/mpvEhwydqloDfqnhtZwC1ps8luu/iKRss1TQ8rdghTyRIoEu9Cvyjm5w2qkYQVyEJK7UkMr0K6XKNSlRSXJMiME1NcNJbMIbBMmBAuhAUf+jZRz0kaZ91bkEPuagGtkEGKxlHt4twwSiM+w2HUc64hUUu6q5HJMkQAq1AXpmowrby+MwKTLA86og2uYDdCyJalRVdXXdDTa4Shbs+YIko2WlW6x7JBsmfCGOCWfMo8lu4r8HU64PEkVxkn8To9f9p78MjZ88u0TzLQaeDkHEswa7nDUEsGKhTfiHJdghkaw9xTD40WeOE0I/v2JY5hnsgE4p7cBNOSDMWxpAzD7x9kAmreIPBHH3QAsjWABkhfwKYkQhU8Ovs2HM+bbsCu62Q1BYzb3hH3pZts7HLVEt2YcqxNBo9fj7pSEPluKuf+MYu6BFBsOdjhqiWJO47ZtdSJp9JIixyLl0VcpxmIMTbc+SgLsq3HQrEr6brvkc/ckn7OefF9kmuOsB+7zTDN2OGqLaVoFrgum6VW2M2Tavle3wVGYZjb3tKa42pdpo25EmtEUg3WRtrPO0Y2o0wtVibw19FflKsIiiTdc0hKBwrjrk7eLk9KyWmKV3Zn2N9bGOSpN8tO5QpM09lH3YqyT62Bb+lzj1pH0OSz82l4b6bik0CtCHIfsA9fuhP8hQQaN+5BW1Dl0z9AcfZogtRLSVozhtfzDRda9/N+Kdf8X</diagram></mxfile>
```

Comment:
`set deletion timestamp` should probably be initiated from `k8s apiserver` rather than `queue`

Reply:
good point